libr/bin/bfile.c

```diff
@@ -447,12 +447,17 @@ static bool xtr_metadata_match(RBinXtrData *xtr_data, const char *arch, int bits
 	if (!xtr_data->metadata || !xtr_data->metadata->arch) {
 		return false;
 	}
-
-	char *iter_arch = xtr_data->metadata->arch;
+	const char *iter_arch = xtr_data->metadata->arch;
 	int iter_bits = xtr_data->metadata->bits;
 	return bits == iter_bits && !strcmp (iter_arch, arch) && !xtr_data->loaded;
 }
 
+R_IPI RBinFile *r_bin_file_new_from_buffer(RBin *bin, const char *file, RBuffer *buf, ut64 file_sz, int rawstr, ut64 baseaddr, ut64 loadaddr, int fd, const char *pluginname, ut64 offset) {
```

Comment:
ok, i guess this wrapper is just for now, right?

Reply:
yes. we must kill this when the PR gets merged. but at least this way we can kill all the public _bytes apis in here